massa-consensus-worker/src/tests/scenario_block_creation.rs

```diff
@@ -73,7 +73,50 @@ async fn test_genesis_block_creation() {
     .await;
 }
 
-// implement test of issue !424.
+/// /// See the test removed at https://gitlab.com/massalabs/massa-network/-/merge_requests/381/diffs#a5bee3b1b5cc9d8157b6feee0ac3e775aa457a33_544_539
+///
+/// **NOTE: that test is expected to fail 1 / 1000 times**
+///
+///
+/// ### Context
+///
+/// * price per roll = 1000
+/// * periods per cycle = 30 000
+/// * t0 = 500ms
+/// * lookback = 2
+/// * thread count = 2
+/// * delta f0 = 3
+/// * genesis timestamp = now - t0 * periods per cycle * 3 - 1000
+/// * block reward = 0
+/// * fee = 0 for every operation
+/// * addr 1 has 1 roll and 0 coins
+/// * addr 2 is in consensus and has 0 roll and 1000 coins
+///
+/// ### Initialization
+/// Following blocks are sent through a protocol event to consensus right at the beginning. They all have best parents as parents.
+/// * block at slot(1,0) with operation addr 2 buys 1 roll
+/// * block at slot( period per cycle, 0)
+/// * block at slot( period per cycle, 1)
+/// * block at slot( period per cycle + 1, 0)
+/// * block at slot( period per cycle + 1, 1)
+/// * block at slot( period per cycle + 2, 0)
+/// * block at slot( period per cycle + 2, 0)
```

Comment:
Did you mean ... ?
```suggestion
/// * block at slot( period per cycle + 2, 1)
```

Reply:
I don't remember x)
I just copy pasted the scenario as described in the issue
Maybe it's an issue with the test ?